### PR TITLE
[test] Use 5ULP of FP16 range for MPS computes on FP16. Fixes #65.

### LINKED ISSUE #65: Which would be the right accuracy for Web ML performance score?
Currently we use 1e-6f as accuracy for Web ML performance score, while Android NN CTS uses 1e-5f or 5.0f * 0.0009765625f for relaxed mode from **executeWithCompilation** function in https://android.googlesource.com/platform/frameworks/ml/+/master/nn/runtime/test/GeneratedUtils.cpp
```cpp

  Line 89:     void executeWithCompilation(Model* model, Compilation* compilation,
  Line 90:                                 std::function<bool(int)> isIgnored,
  Line 91:                                 std::vector<MixedTypedExample>& examples,
  Line 92:                                 std::string dumpFile) {
  ...
  Line 101:         // If in relaxed mode, set the error range to be 5ULP of FP16.
  Line 102:         float fpRange = !model->isRelaxed() ? 1e-5f : 5.0f * 0.0009765625f;
  ...
  Line 146:             compare(filteredGolden, filteredTest, fpRange);
```
Which would be the right accuracy for Web ML, 1e-6f or other to be defined?

@halton @huningxin what are your opinions, thanks.

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin PTAL, thanks.

### CODE REVIEW

File: test/utils.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const nn = navigator.ml.getNeuralNetworkContext();
 let options = {};
 let prefer = nn.PREFER_FAST_SINGLE_ANSWER;
+let episilonCTS = 1e-5;
```
Comment (huningxin) [inline test/utils.js]: use `const`

File: test/utils.js [hunk 2]
```diff
@@ -37,6 +38,7 @@ function setOptions() {
     } else if (backend == "mps") {
       // use PREFER_SUSTAINED_SPEED for MacOS MPS backend
       prefer = nn.PREFER_SUSTAINED_SPEED;
+      episilonCTS = 5.0 * 0.0009765625;
```
Comment (huningxin) [inline test/utils.js]: you may define another `const` for the 5ULP and use it here.

Comment (huningxin) [reply test/utils.js]: And please add comments of where it comes from.

**Comment by BruceDai** [comment]
@huningxin Thanks for first review, I have updated code, PTAL.

**Comment by huningxin** [comment]
Looks good. Thanks!

**Pull request merged**